ksqldb-engine/src/main/java/io/confluent/ksql/analyzer/Analysis.java

```diff
@@ -89,8 +88,8 @@ ResultMaterialization getResultMaterialization() {
     return resultMaterialization;
   }
 
-  void addSelectItem(final Expression expression, final ColumnName alias) {
-    selectExpressions.add(SelectExpression.of(alias, expression));
+  void addSelectItem(final SelectItem selectItem) {
```

Comment:
Previously, we captured the _resolved_ select items - now we just capture the unresolved `SelectItem`